jf_agent/jira_download.py

```diff
@@ -292,6 +292,48 @@ def download_worklogs(jira_connection, issue_ids):
     return {'existing': updated, 'deleted': []}
 
 
+# Returns an array of CustomFieldOption items
+@diagnostics.capture_timing()
+@agent_logging.log_entry_exit(logger)
+def download_customfieldoptions(jira_connection):
+    print('downloading jira custom field options... ', end='', flush=True)
+    try:
+        meta = jira_connection.createmeta(projectIds=None, expand='projects.issuetypes.fields')
+    except JIRAError:
+        agent_logging.log_and_print(
```

Comment:
Should probably just be `print` to be consistent with other error handlers.

We're careful printing things to the logger in the agent since we want to make sure we don't print anything that could contain customer-sensitive info -- e.g., names of their things. This message that you have doesn't do that, but probably best to stay consistent with the other code anyway.

Reply:
:+1: